cms/templates/settings.html

```diff
@@ -319,7 +319,7 @@ <h3 id="heading-entrance-exam">${_("Entrance Exam")}</h3>
                             <label for="entrance-exam-enabled">${_("Require students to pass an exam before beginning the course.")}</label>
                         </div>
                          <div class="div-grade-requirements" hidden="hidden">
-                          <p><span class="tip tip-inline">${_("You can now view and author your course entrance exam from the ")}<a href='${course_handler_url}'>${_("Course Outline")}</a></span></p>
+                          <p><span class="tip tip-inline">${_("You can now view and author your course entrance exam from the {link_start}Course Outline{link_end}.").format(link_start="<a href='{}'>".format(course_handler_url), link_end="</a>")}</span></p>
```

Comment:
@mattdrayer : this is a line you did. In the future, please do **not** separate parts of a sentence - it causes context to be lost and makes translation very difficult.